### PR TITLE
added code comments

### PR DESCRIPTION
**Pull request by guonaihong**

### CODE REVIEW

**Review by kevwan — APPROVED** [review]

File: zrpc/internal/balancer/p2c/p2c.go [hunk 1]
```diff
@@ -121,6 +121,28 @@ func (p *p2cPicker) buildDoneFunc(c *subConn) func(info balancer.DoneInfo) {
 		if td < 0 {
 			td = 0
 		}
+
+		// The function curve of y = x^(-x) is as follows.
```
Comment (kevwan) [inline zrpc/internal/balancer/p2c/p2c.go:125]: Replace this text curve with a chart link?

Comment (guonaihong) [reply zrpc/internal/balancer/p2c/p2c.go:125]: Ok, under which repository of the zeromicro account should the image be placed?

Comment (kevwan) [reply zrpc/internal/balancer/p2c/p2c.go:125]: You can put it in zero-doc

Comment (guonaihong) [reply zrpc/internal/balancer/p2c/p2c.go:125]: https://github.com/zeromicro/zero-doc/pull/179

File: zrpc/internal/balancer/p2c/p2c.go [hunk 2]
```diff
@@ -121,6 +121,29 @@ func (p *p2cPicker) buildDoneFunc(c *subConn) func(info balancer.DoneInfo) {
...
+		//         *
+		//          *
+		//           **
+		//             *
+		//              ***
+		//                 **
+		//                   ****
+		//                       ******
+		//                             *********************
+		//                                                  *
```
Comment (kevwan) [inline zrpc/internal/balancer/p2c/p2c.go:143]: Please remove this character chart.

Comment (guonaihong) [reply zrpc/internal/balancer/p2c/p2c.go]: I have deleted this character chart

**Pull request merged**